src/query/query_hash_scan.c

```diff
@@ -2504,56 +2503,72 @@ static FHS_RESULT
 fhs_insert_to_dk_bucket (THREAD_ENTRY * thread_p, FHSID * fhsid_p, VPID * next_bucket_vpid, void *key_p,
 			 TFTID * value_p)
 {
-  RECDES bucket_recdes, old_bucket_recdes;
+  RECDES bucket_recdes = RECDES_INITIALIZER, first_bucket_recdes = RECDES_INITIALIZER, last_bucket_recdes =
+    RECDES_INITIALIZER;
   PGSLOTID tmp_slot;
   int success;
-  FHS_DK_BUCKET_HEADER *dk_bucket_header_p;
-  VPID tmp_bucket_vpid, cur_bucket_vpid;
-  PAGE_PTR dk_bucket_page_p = NULL;
+  FHS_DK_BUCKET_HEADER *first_dk_bucket_header_p = NULL, *last_dk_bucket_header_p = NULL;
+  VPID last_bucket_vpid, cur_bucket_vpid;
+  PAGE_PTR first_dk_bucket_page_p = NULL;
+  PAGE_PTR last_dk_bucket_page_p = NULL;
   PAGE_PTR new_dk_bucket_page_p = NULL;
   VPID null_vpid = { NULL_VOLID, NULL_PAGEID };
```

Comment:
It's a minor correction.
I recommend it as below.
  VPID null_vpid = { NULL_PAGEID, NULL_VOLID };  
or 
  VPID null_vpid = VPID_INITIALIZER;

Reply:
I fixed it.